### PR TITLE
ElasticDL Client: Implement "zoo init | build | push" command.

### PR DESCRIPTION
**Pull request by brightcoder01**
Relate #1918

### CODE REVIEW

File: elasticdl_client/api.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+{% endif %}
+"""
+    template = Template(tmpl_str)
+    docker_file_content = template.render(
+        BASE_IMAGE=args.base_image,
+        EXTRA_PYPI_INDEX=args.extra_pypi_index,
+        CLUSTER_SPEC_NAME=cluster_spec_name,
+    )
+
+    with open("./Dockerfile", mode="w+") as f:
```
Comment (skydoorkai) [inline elasticdl_client/api.py:62]: nitpicking: `mode="w" ` is enough.

File: elasticdl_client/common/args.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        type=str,
+        help="The file that contains user-defined cluster specification,"
+        "the file path can be accessed by ElasticDL client.",
+        default="",
+    )
+
+
+def add_zoo_build_arguments(parser):
+    parser.add_argument(
+        "path", type=str, help="The path where the build context locate."
```
Comment (skydoorkai) [inline elasticdl_client/common/args.py:39]: "The path where the build context locate." -> 
The path where the build context locates.

**Review by wangkuiyi — APPROVED** [review]
LGTM.

We could fix comments in a future PR.

File: elasticdl_client/common/args.py (hunk 2 above)
Comment (brightcoder01) [reply elasticdl_client/common/args.py:39]: Will resolve in the next PR.

File: elasticdl_client/api.py (hunk 1 above)
Comment (brightcoder01) [reply elasticdl_client/api.py:62]: Will resolve in the next PR.

**Pull request merged**